Year 2 updates and eligibility changes (#3584)
DFE-Digital/get-into-teaching-app: app/views/content/non-uk-teachers/get-an-international-relocation-payment.md

inset_text:
  applications-open:
    text: |-
      <p>Teachers and salaried trainees who started their job or course in September 2023 <a href="https://get-a-teacher-relocation-payment.education.gov.uk/"> can now apply for an international relocation payment (IRP)</a>. You must apply by 31 October 2023.</p>
      
      <p>If you are starting a teaching job after the end of October 2023, or have missed the 2023 deadline, you can express your interest in the IRP by emailing us at IRP.ExpressInterest@education.gov.uk. We will email you when applications re-open in 2024.</p> 
      

>       <p>If you are a trainee on a fee-paying course, you do not need to apply. <a href="/non-uk-teachers/get-an-international-relocation-payment#how-trainee-teachers-can-get-the-international-relocation-payment">Learn more about the process for fee paying courses</a>.</p>

Author: I think not cos because the expression of interest stuff doesn't apply to trainees...

All eligible applicants will receive a single payment of £10,000 from the Department for Education (DfE) around the end of their first term training or working in England. The IRP is a grant – you will not have to pay this money back. You can only receive the IRP once.

> [Learn more about teaching in England if you trained outside the UK](/non-uk-teachers/teach-in-england-if-you-trained-overseas).

Author: Um I added them because I think you've found in UR that those related content links are often missed? I think they're OK where they are :)

* 50% general or combined science (must include the physics element) and 50% maths 

Examples of eligible languages commonly taught in English state secondary schools include: 

> * Ancient languages (such as Latin) 

Author: I have a feeling you need to use line breaks to turn the asterisks into bullets but I will see what happens!

### 2. Type of visa for teachers

> You must come to England on one of the visas listed here. Most teachers will need a Skilled worker visa. 

Reviewer: ```suggestion
You must come to England on one of the visas listed here. Most teachers will need a Skilled Worker visa. 
```

You will need a teaching job to apply for the IRP. [Learn more about finding a job and applying for your visa](/non-uk-teachers/teach-in-england-if-you-trained-overseas).

If you are starting a teaching job after the end of October 2023, or have missed the 2023 deadline, you can express your interest in the IRP by emailing us at IRP.ExpressInterest@education.gov.uk. We will email you when applications re-open in 2024.

## Criteria for trainee teachers (fee-paying and salaried) 

> As a trainee teacher, to be eligible for the international relocation payment (IRP), you must meet all 4 of the following requirements. The eligibility requirements are the same whether you pay a fee for your training course, or are paid a salary to work while you train. However, the application process for the IRP is [different for salaried trainees](#how-trainee-teachers-can-get-the-international-relocation-payment).

Reviewer: This link probably needs a bit more context to make it accessible. 
```suggestion
As a trainee teacher, to be eligible for the international relocation payment (IRP), you must meet all 4 of the following requirements. The eligibility requirements are the same whether you pay a fee for your training course, or are paid a salary to work while you train. However, [the application process is different for salaried trainees](#how-trainee-teachers-can-get-the-international-relocation-payment).
```

### 1. Teaching subject for trainees

You must have taken up a place on a teacher training course in England leading to [qualified teacher status (QTS)](https://www.gov.uk/guidance/qualified-teacher-status-qts) for one of the following academic years:

* the 2023 to 2024 academic year
* the 2024 to 2025 academic year

Your teacher training provider must be accredited by the UK government, and you must be training to teach one of the following subjects at secondary level (teaching children aged 11 to 16 or 11 to 18):

* French 

* German 

* Spanish 

* a combination of French, German or Spanish 

* French, German or Spanish combined with another subject, as long as languages make up at least 50% of the course content 

* French, German or Spanish combined with English language, as long as non-English languages make up at least 50% of course content

* physics

* physics combined with another subject, as long as physics makes up at least 50% of the course content

> You can search for [postgraduate teacher training courses](https://www.gov.uk/find-postgraduate-teacher-training-courses) accredited by the UK government leading to the award of QTS. You will still be eligible for the international relocation payment if your course leads to PGCE and QTS or PGDE and QTS.

Reviewer: ```suggestion
You can [search for postgraduate teacher training courses](https://www.gov.uk/find-postgraduate-teacher-training-courses) accredited by the UK government leading to the award of QTS. You will still be eligible for the international relocation payment if your course leads to PGCE and QTS or PGDE and QTS.
```

* the content of your teacher training course meets the 50% requirement

### 2. Degree requirements for trainees

> If you are starting your teacher training course in or after September 2024, you must have studied for your undergraduate (bachelor’s) degree at a [university recognised](https://www.enic-naric.net/page-recognised-heis) by the European Network of Information Centres (ENIC).

Reviewer: Need to tweak this link text to make it more accessible. 
```suggestion
If you are starting your teacher training course from September 2024, you must have studied for your undergraduate (bachelor’s) degree at a university recognised by the European Network of Information Centres (ENIC). You can check if your university is recognised on the [ENIC website](https://www.enic-naric.net/page-recognised-heis).
```

Author: We decided to remove this as it's not quite accurate.

> If you are on a fee-paying teacher training course, you will not have to pay tax on the international relocation payment.  

Reviewer: ```suggestion
If you are on a fee-paying teacher training course, you will not have to pay tax on the international relocation payment (IRP).  
```

> If you are a teacher, or a trainee on a salaried course, the Department for Education will pay the UK tax owed on the international relocation payment (IRP) to ensure you receive the full £10,000, as long as your overall earnings do not exceed £50,270 in the year you receive the IRP. You may have to pay higher rate [UK Income Tax](https://www.gov.uk/browse/tax/income-tax) and [UK National Insurance](https://www.gov.uk/national-insurance) on any earnings which take you over £50,270.

Reviewer: ```suggestion
If you are a teacher, or a trainee on a salaried course, the Department for Education will pay the UK tax owed on the IRP to ensure you receive the full £10,000, as long as your overall earnings do not exceed £50,270 in the year you receive the IRP. You may have to pay higher rate [UK Income Tax](https://www.gov.uk/browse/tax/income-tax) and [UK National Insurance](https://www.gov.uk/national-insurance) on any earnings which take you over £50,270.
```
